src/main/java/com/iota/iri/network/Node.java

```diff
@@ -103,6 +113,7 @@ public void init() throws Exception {
         parseNeighborsConfig();
 
         executor.submit(spawnBroadcasterThread());
+        executor.submit(spawnRequesterThread());
```

Comment:
Its the thread that works through the requestQueue rather than specifically requesting selective transactions - dunno if another name would make it easier to understand - but i don't really mind how its called